src/main/java/seedu/address/model/session/Interval.java

```diff
@@ -46,6 +52,10 @@ public LocalDateTime getStart() {
         return this.start;
     }
 
+    public String getStartDay() {
+        return this.start.format(SIMPLE_DATE_TIME_PATTERN_FORMATTER);
+    }
+
```

Comment:
Is it better to just have one function `String getFormattedStartDateTime (DateTimeFormatter formatter)` ?